utils/build/docker/dotnet/parametric/ApmTestApi.csproj

```diff
@@ -11,8 +11,8 @@
 
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
-    <PackageReference Include="Datadog.Trace" Version="*" Condition="'$(Configuration)' != 'Release'" />
-    <Reference Include="/opt/datadog/netcoreapp3.1/Datadog.Trace.dll" Condition="'$(Configuration)' == 'Release'" />
+
+    <PackageReference Include="Datadog.Trace" Version="*" />
```

Comment:
Don't we want to use `/opt/datadog/netcoreapp3.1/Datadog.Trace.dll` instead for the unpublished builds (`dev` vs `prod`)?

Reply:
I let this one for .NET guild